command/login.go

```diff
@@ -112,11 +112,11 @@ func (c *LoginCommand) Flags() *FlagSets {
 	})
 
 	f.BoolVar(&BoolVar{
-		Name:    "no-print",
-		Target:  &c.flagNoPrint,
+		Name:    "print-token",
```

Comment:
are we concerned with making breaking changes to the cli?

Reply:
Yeah that is a good question @averche idk if you know the protocol on that? This previous flag I think doesn't have a ton of value anymore but idk if it needs to just be deprecated and left around for a while?

Reply:
Agreed, I think we should definitely still support this flag, otherwise a lot of the CI workflows (which are likely to use it) will break with the following:

```sh
flag provided but not defined: -no-print
```

We could consider deprecating it here instead or just use it to avoid printing `"Success! You are now authenticated..."`